disco/extensions/upgrade_simulation/upgrades/upgrade_parameters.toml

```diff
@@ -0,0 +1,57 @@
+[thermal_upgrade_params]
```

Comment:
We need one set of defaults. I made some Pydantic models. Should we use the default there or should we make those models load these values?

Reply:
Yeah, it's duplication. We do need a `toml` file for user customization and input via CLI, maybe make model values loaded from the toml file.